app/models/active_storage/attachment.rb

```diff
@@ -12,8 +12,6 @@ class ActiveStorage::Attachment < ActiveRecord::Base
   belongs_to :record, polymorphic: true, touch: true
   belongs_to :blob, class_name: "ActiveStorage::Blob"
 
-  after_commit :analyze_blob_later, on: :create
```

Comment:
Would it be better if we override this method in our code and make it a noop, rather than modifying activestorage's functionality?

Reply:
Well I don't think we should override this method. This method maybe used in future if need analysis on blobs is required. If so we can then add this callback back. Also if keep things intact we can add/remove analyzer as required ie. http://edgeapi.rubyonrails.org/classes/ActiveStorage/Blob/Analyzable.html

Reply:
Let's also remove method `analyze_blob_later`

Reply:
We can still keep it. If we need to manually analyze some blobs? what do you think?

Reply:
blob analyze doesn't work, no sense of leaving method there. later when we need it, we will add it back.